examples/gulp/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+var gulp = require('gulp'),
+    connect = require('gulp-connect'),
+    metalsmith = require('metalsmith'),
+    markdown = require('metalsmith-markdown'),
+    permalinks = require('metalsmith-permalinks'),
+    templates = require('metalsmith-templates'),
+    watch = require('metalsmith-watch');
+
+    gulp.task('serve', function() {
+        connect.server();
+    });
```

Comment:
Was the intent here to serve the entire project directory, including the source folder, gulpfile, etc? Wouldn't it make more sense to just serve the `build/` directory?